back/src/companies/resolvers/CompanyPublic.ts

```diff
@@ -3,16 +3,19 @@ import { CompanyPublicResolvers } from "../../generated/graphql/types";
 
 const companyPublicResolvers: CompanyPublicResolvers = {
   transporterReceipt: async parent => {
-    const transporterReceipt = await prisma.company
+    return await prisma.company
```

Comment:
Micro détail, mais `return await` est redondant à priori en dehors d'un bloc try/catch.

```suggestion
    return prisma.company
```